Puller.java

```diff
@@ -17,46 +18,107 @@
 import java.util.Iterator;
 public class Puller {
 	
-
-
+	/*
+	 * Hey buddy,
+	 * 	I set the currentRow in the 2nd while loop to get the cell values
+	 * 	I added a few more lines, mostly just print statements and comments
+	 * 	You should run it to see what it outputs.
+	 * 
+	 * --Chris "Big Data" Mendoza
+	 */
+	
             private static final String FILE_NAME = "C:\\Users\\syntel\\Downloads\\MockData.xlsx";
 
             @SuppressWarnings("deprecation")
```

Comment:
Good enough for now, but should we really be using deprecated stuff?

Reply:
I wasnt sure what that meant. For now it works good enough for the prototype though, and we can fix that layer